app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -8,29 +8,55 @@ def initialize(identity)
   end
 
   def user_info
-    user = identity.user
-
-    {
+    info = {
       sub: identity.uuid,
       iss: root_url,
-      email: user.email,
+      email: identity.user.email,
```

Comment:
Yeah I can see that. I think not violating the law of demeter is also a good goal, and in this case would require delegation. But happy to keep as-is if others prefer demeter violations to delegation. https://devblast.com/b/rails-delegate-dont-break-the-law-of-demeter